resources/Microsoft.Windows.Settings/source/Classes/010.SettingsBase.ps1

```diff
@@ -0,0 +1,114 @@
+<#
```

Comment:
Can I ask where the numerical prefixes are coming from on these files? I see other classes like this in DSC community repos and are aware that they may be coming from Sampler templates, but I am not finding a clear explanation or a way to interpret them.